src/components/InternalLink/index.tsx

```diff
@@ -28,9 +28,17 @@ export default function InternalLink({ href, children }) {
         const filterKey = filterKeys[filterKind];
         if (href.includes('#')) {
           const hrefParts = href.split('#');
-          href = `${hrefParts[0]}/q/${filterKind}/${filterKey}#${hrefParts[1]}`;
+          if (hrefParts[0].endsWith('/')) {
+            href = `${hrefParts[0]}q/${filterKind}/${filterKey}#${hrefParts[1]}`;
+          } else {
+            href = `${hrefParts[0]}/q/${filterKind}/${filterKey}/${hrefParts[1]}`;
```

Comment:
missing # on this line `${hrefParts[0]}/q/${filterKind}/${filterKey}#${hrefParts[1]}`

Reply:
Thank you!!